src/@orb.yml

```diff
@@ -50,7 +55,8 @@ jobs:
     docker:
       - image: gitguardian/ggshield:<<parameters.tag>>
     environment:
-      CIRCLE_RANGE: <<parameters.base_revision>>...<<parameters.revision>>
+      NUMBER_COMMITS_GH: <<parameters.number_commits_gh>>
+      NUMBER_COMMITS_GL: <<parameters.number_commits_gl>>
```

Comment:
Would have been nice to expose only one variable, but I assume it's not possible to do operations like `max` or `+` here?

Reply:
It seems possible if I do something like 

```
GITHUB_COUNT="<<pipeline.trigger_parameters.github_app.total_commits_count>>"
GITLAB_COUNT="<<pipeline.trigger_parameters.gitlab.total_commits_count>>"
MAX_COUNT=$(( GITHUB_COUNT > GITLAB_COUNT ? GITHUB_COUNT : GITLAB_COUNT ))
echo "export CIRCLE_COMMIT_COUNT=$MAX_COUNT" >> $BASH_ENV
```

In `run`, before the ggshield secret scan command

Do you think it's worth it ?

Reply:
It's a bit hackish, but that is what happens with shell scripting :). I think it's worth it because it means we don't have to make a new ggshield release if we ever find a way to get the BitBucket count, or whatever ci-host-of-the-day is added to CircleCI in the future.

Reply:
Hi @agateau-gg, I am tackling this issue since I have some time now but I am really struggling to make this work.
I've tried a lot of solutions (you can see them in the commits of [this branch](https://github.com/amascia-gg/test_circle_ci/tree/amascia/test-nb-commits-var)) but I can't find a way to fetch both values (for GitHub and GitLab). Can we have a look together if you have some time please?